packages/forklift-console-plugin/src/modules/Overview/views/overview/components/ShowTheWelcomeCardButton.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useCreateOverviewStateContext } from 'src/modules/Overview/utils/helpers/OverviewStateContextProvider';
+import { setWelcomeStateByUserSettings } from 'src/modules/Overview/utils/helpers/OverviewUserSettings';
+
+import { Label } from '@patternfly/react-core';
+
+export const ShowTheWelcomeCardButton: React.FC = () => {
```

Comment:
```suggestion
export const ShowWelcomeCardButton: React.FC = () => {
```

we only have one welcome card